db/migrate/20231216101427_add_support_user_enabled_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddSupportUserEnabledToUsers < ActiveRecord::Migration[7.1]
+  def change
+    add_column :users, :support_user_enabled, :boolean, default: false
```

Comment:
I agree -- Originally I had it with just `support_user`, and for the the school table, I just had `claims` and `placements` instead of `claims_enabled` and `placements_enabled`. Tbh, I was following the pattern on the erb.

Reply:
Fair play! You make a good point about the `claims_enabled` and `placements_enabled` too.

I guess, conceptually, I've been thinking of these differently:

- A user is _identified by_ the service they belong to. So they are either a "Claims User" or a "Placements User". And if the human has access to both services, they are represented as two records.
- A School is identified by its URN. There is only ever one School. And that School can be onboarded into each of the services independently.

I expect this thinking is what's led me to model the fields differently in the ERD.

I'm not fixed on this view, but for whatever reason they seem like meaningful differences in my mind right now 🤔